src/components/account/company/appsco-account-devices-item.html

```diff
@@ -0,0 +1,187 @@
+
+                <template is="dom-if" if="[[ _approved ]]">
+                    <paper-button on-tap="_onDisapproveAction">Disapprove</paper-button>
+                </template>
+            </div>
+        </div>
+
+    </template>
+
+    <script>
+        Polymer({
+
+            is: 'appsco-account-devices-item',
+
+            properties: {
+
+                /**
+                 * Group that is to be rendered
+                 */
+                device: {
+                    type: Object,
+                    value: function() {
+                        return {};
+                    }
+                },
+
+                /**
+                 * Account
+                 */
+                account: {
+                    type: Object,
+                    value: function () {
+                        return {};
+                    }
+                },
+
+                /**
+                 * Indicates if devices should be in preview mode rather then full detailed view.
+                 */
+                preview: {
+                    type: Boolean,
+                    value: false,
+                    reflectToAttribute: true
+                },
+
+                _approved: {
+                    type: Boolean,
+                    computed: "_computeApproved(device)"
+                },
+
+                /**
+                 * Animation for loader appearance.
+                 */
+                animationConfig: {
+                    value: function() {
+                        return {
+                            'entry': {
+                                name: 'fade-in-animation',
+                                node: this,
+                                timing: {
+                                    duration: 200
+                                }
+                            },
+                            'exit': {
+                                name: 'fade-out-animation',
+                                node: this,
+                                timing: {
+                                    duration: 100
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+
+            behaviors: [
+                Polymer.NeonAnimationRunnerBehavior
+            ],
+
+            attached: function() {
+                this.style.display = 'inline-block';
+                this.playAnimation('entry');
+            },
+
+            _computeApproved: function(device) {
+                return device.approved;
+            },
+
+            _onApproveAction: function(event) {
+                this.device.approved = true;
+                event.stopPropagation();
+
+                /**
+                 * Fired when `Approve` action is tapped.
+                 *
+                 * @event device-approved
+                 * @param {Object} group
+                 */
+                this.fire('device-approved', {
```

Comment:
Event should be in form of action: approve-device. 
device-approved should be used when action is done.
The same for disapprove.